### ISSUE TITLE
Create a contact form

### ISSUE THREAD

**Issue by duleorlovic** [trkin/kindergarten-exchange#4]
This form should be public.
It requires: name (string), email (string) (populated if it is already logged in) and question (text) field.
Write a test that the email is sent.

**Comment by duleorlovic** [comment]
I think you can use Form object, just plain ruby class with `include ActiveModel::Model`
eg https://github.com/duleorlovic/rails_helpers_and_const/blob/main/app/forms/registration_form.rb
```
# app/forms/contact_form.rb
class ContactForm
  include ActiveModel::Model
  FIELDS = %i[name email question]
  attr_accessor *FIELDS

  validates *FIELDS, presence: true
def save
    return false unless valid?

    # here you can send the email

    true
  end
end
```

**Comment by DeVuDeveloper** [comment]
Hello @Orlovic982, can I work on this task, please?

**Comment by Orlovic982** [comment]
Yes,of course. I dont have much time at the moment, but I will catch you guys later

**Comment by DeVuDeveloper** [comment]
https://github.com/trkin/kindergarten-exchange/pull/41
Ready for review. It would be nice if it can be done today. One before this is waiting too. Details in PR summary.

**Comment by DeVuDeveloper** [comment]
https://github.com/trkin/kindergarten-exchange/pull/41
All required changes are made.
@ more Pull requests are waiting for review.

**Comment by DeVuDeveloper** [comment]
https://github.com/trkin/kindergarten-exchange/pull/41
Required changes made.

**Comment by duleorlovic** [comment]
@vesnalesevic can you work on contact form ?
You can learn about Form objects on https://openai.com/blog/openai-api
